cmd/gmnhg/main.go

```diff
@@ -305,6 +336,7 @@ func main() {
 				for _, dir := range dirs {
 					topLevelPosts[dir] = append(topLevelPosts[dir], p)
 				}
+				topLevelPosts[""] = append(topLevelPosts[""], p)
```

Comment:
This reads hackish, and probably reliant on the hash algorithm Go maps use. Maybe we should make this an extra map after all.

Reply:
I was unsure of that too. It would require a couple of minor changes elsewhere, but I wonder if prefixing all paths with "/" would be a good solution. Handling the "root" path in the same way as the "branch" paths simplifies things somewhat and it makes intuitive sense, as they are all part of the same path tree.

Reply:
Just wanted to add that one potential use for this is if someone wants to add a block of "recently changed pages" for their whole site to their main index. Adding the "root" path to PostData makes this easier.

Reply:
True, starting paths with single slash should be fine.